Assets/Plugins/LeapMotion/Internal/VRVisualizer/ScenesResources/Scripts/VisualizerManager.cs

```diff
@@ -93,24 +101,39 @@ void Update()
       m_trackingText.text = "Tracking Mode: ";
       m_trackingText.text += (m_controller.IsPolicySet(Controller.PolicyFlag.POLICY_OPTIMIZE_HMD)) ? "Head-Mounted" : "Desktop";
 
-
-      // In Desktop Mode
-      if (m_PCVisualizer.activeInHierarchy)
+      if (m_controller.IsPolicySet(Controller.PolicyFlag.POLICY_OPTIMIZE_SCREENTOP))
       {
-        if (m_controller.IsPolicySet(Controller.PolicyFlag.POLICY_OPTIMIZE_HMD))
+        m_trackingText.text = String.Format(
+          "Tracking Mode: Screen-Top (Press '{0}' to switch to desktop mode)",
+          keyToSwitchViewMode);
+        if (Input.GetKeyDown(keyToSwitchViewMode))
```

Comment:
'keyToSwitchViewMode' is currently named 'keyToToggleHMD' in this commit so these lines throw errors. I expect renaming keyToToggleHMD to keyToSwitchViewMode makes most sense?